### PR TITLE
Switch to plausible over Google Analytics

### PR DESCRIPTION
**Pull request by alexandria-gfsc**
This switches to Plausible over Google Analytics and (hopefully) correctly alters the Privacy Disclaimer

### CODE REVIEW

**Comment by alexandria-gfsc** [comment]
Fixed #855

**Comment by alexandria-gfsc** [comment]
lol there are tests that depend on google analytics???

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -14,6 +14,7 @@
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
 
     <%= render "layouts/meta" %>
+    <%= render "layouts/plausible" %>
```
Comment (katjam) [inline app/views/layouts/application.html.erb:17]: Not laughing. It was late.

File: app/views/pages/privacy.html.erb [hunk 2]
```diff
@@ -8,7 +8,7 @@
...
+<p>PlaceCal.org’s Privacy Policy does not apply to Plausible, which we use to track visits. Thus, we are advising you to consult the respective Privacy Policies of Plausible for more detailed information. It may include their practices and instructions about how to opt-out of certain options. Plausible's Data Policy is here: https://plausible.io/data-policy</p><p>Plausible does not use Cookies to track your data, nevertheless, you may choose to disable cookies through your individual browser options. To know more detailed information about cookie management with specific web browsers, it can be found at the browsers’ respective websites. </p>
```
Comment (katjam) [inline app/views/pages/privacy.html.erb:11]: :grin:

**Review by katjam — APPROVED** [review]
Bravo!

**Pull request merged**